### PR TITLE
[DOC] Add quantized inception updated documentation.

### PR DESCRIPTION
**Pull request by yassineAlouini**
In this PR, I add the updated quantized inception documentation.

### CODE REVIEW

File: docs/source/models/inception_quant.rst [hunk 1]
```diff
@@ -0,0 +1,24 @@
+Quantized Inception
+===================
```
Comment (oke-aditya) [inline docs/source/models/inception_quant.rst:2]:
```suggestion
=====================
```

File: torchvision/models/quantization/inception.py [hunk 2]
```diff
@@ -219,10 +219,26 @@ def inception_v3(
     GPU inference is not yet supported
 
     Args:
-        weights (Inception_V3_QuantizedWeights or Inception_V3_Weights, optional): The pretrained
-            weights for the model
-        progress (bool): If True, displays a progress bar of the download to stderr
-        quantize (bool): If True, return a quantized version of the model
+        weights (:class:`~torchvision.models.quantization.Inception_V3_QuantizedWeights` or :class:`~torchvision.models.quantization.Inception_V3_Weights`, optional): The pretrained
```
Comment (NicolasHug) [inline torchvision/models/quantization/inception.py:222]:
```suggestion
        weights (:class:`~torchvision.models.quantization.Inception_V3_QuantizedWeights` or :class:`~torchvision.models.Inception_V3_Weights`, optional): The pretrained
```

File: torchvision/models/quantization/inception.py [hunk 3]
```diff
@@ -219,10 +219,26 @@ def inception_v3(
...
+            :class:`~torchvision.models.quantization.Inception_V3_QuantizedWeights` below for
+            more details, and possible values. By default, no pre-trained
+            weights are used.
+        progress (bool, optional): If True, displays a progress bar of the download to stderr.
+            Default is True.
+        quantize (bool, optional): If True, return a quantized version of the model.
+            Default is False.
+        **kwargs: parameters passed to the ``torchvision.models.quantization.QuantizableInception3``
+            base class. Please refer to the `source code
+            <https://github.com/pytorch/vision/blob/main/torchvision/models/quantization.inception.py>`_
```
Comment (NicolasHug) [inline torchvision/models/quantization/inception.py:233]:
```suggestion
            <https://github.com/pytorch/vision/blob/main/torchvision/models/quantization/inception.py>`_
```

**Review by NicolasHug — COMMENTED** [review]
Thanks for the PR @yassineAlouini !

Looks great, only a few comments, on top of those from @oke-aditya :)

**Review by NicolasHug — APPROVED** [review]
Thank you @yassineAlouini !

**Pull request merged**